chameleon.py

```diff
@@ -68,20 +90,28 @@ def stop():
 def _get_cherrypy_page_handler(status_code, content_type, content):
     class Foo(object):
         @cherrypy.expose
+        @cherrypy.tools.json_out()
```

Comment:
This kind bugs me.  The tool is supposed to be totally content-type agnostic.  Was there something that wasn't working before that is working now?